lightning/web-app.md

```diff
@@ -330,11 +330,6 @@ $ head -n 3 /home/thunderhub/thunderhub/package.json | grep version
 * Add or edit the following lines, save and exit
 
 ```
-# -----------
-# Server Configs
-# -----------
-NODE_ENV=production
```

Comment:
Why this deletion?

Reply:
Because "production" seems to be the default ENV in Thunderhub and even because "NODE_ENV" variable has never been used in `.env` file shipped with Thunderhub.

Reply:
Ok then, should be delete `:prod` of this line? https://github.com/minibolt-guide/minibolt/pull/76/files#diff-3ae7cf52e17920ec51bbde933c248311c3805b62a5d99c41dfc29e6f17b0a64eR414

Reply:
No, you can leave it, otherwise it doesn't start from dist.

Reply:
> Because "production" seems to be the default ENV in Thunderhub and even because "NODE_ENV" variable has never been used in `.env` file shipped with Thunderhub.

I deleted `NODE_ENV=production` line and despite the service working fine, the output prompt showed me this:

![imagen](https://github.com/minibolt-guide/minibolt/assets/89636253/4fb8a06f-192f-4d04-8ac3-d213b50b2277)

Are you sure that "production" seems to be the default ENV in Thunderhub?

Reply:
I have see this lines:

```
ARG NODE_ENV="production"
ENV NODE_ENV=${NODE_ENV}
```
a couple of times in the docker file and I assumed that it was the default environment. Then, you can ignore my fix if you prefer.

Anyway, why haven't the creator of Thunderbird added the 'NODE_ENV' variable to the .env file?